apps/client/src/app/pages/accounts/transfer-cash-balance/transfer-cash-balance-dialog.html

```diff
@@ -0,0 +1,48 @@
+<form
+  class="d-flex flex-column h-100"
+  [formGroup]="transferCashBalanceForm"
+  (keyup.enter)="transferCashBalanceForm.valid && onSubmit()"
+  (ngSubmit)="onSubmit()"
+>
+  <h1 i18n mat-dialog-title>Transfer Cash Balance</h1>
+  <div class="flex-grow-1 py-3" mat-dialog-content>
+    <div>
+        <mat-form-field appearance="outline" class="w-100">
+          <mat-label i18n>Value</mat-label>
+          <input formControlName="balance" matInput type="number" />
+          <span class="ml-2" matTextSuffix>CHF</span>
```

Comment:
Can you display the currency of the selected `fromAccount` here instead of `CHF`?